examples/scf_properties/atomic_charges/httpx/run.py

```diff
@@ -0,0 +1,138 @@
+                "methodname": "hf",
+                "xc_grid_scheme": "SG2",
+            }
+        },
+        "hf": {
+            "params": {"charge": 0, "multiplicity": 1, "g_convergence": 0.000001},
+        },
+        "scf_properties": {
+            "outputs": [
+                {"type": "atomic_charges", "analysis_method": "mulliken"},
+                {"type": "atomic_charges", "analysis_method": "lowdin"},
+                {"type": "atomic_charges", "analysis_method": "iao"},
+                {"type": "atomic_charges", "analysis_method": "resp"},
+            ],
+        },
+    },
+    "resources": {"gpu_type": gpu_type},
+}
+
+# add metadata only if environment variables exist
+metadata = {}
+workflow_timeout = os.getenv("PM_WORKFLOW_TIMEOUT")
+task_timeout = os.getenv("PM_TASK_TIMEOUT")
+
+if workflow_timeout:
+    metadata["workflow_timeout"] = int(workflow_timeout)
+if task_timeout:
+    metadata["task_timeout"] = int(task_timeout)
+if metadata:
+    job_params["metadata"] = metadata
+
+# submit a SPC workflow using the above configuration
+payload = job_params
+jobname = payload["name"]
+response = client.post("/v0/workflows", json=payload)
+with open(f"{foldername}/{jobname}_submitted.json", "w") as fp:
+    fp.write(json.dumps(response.json()))
+workflow_id = response.json()["id"]
+print(f"Workflow {jobname} submitted with id: {workflow_id}")
+
+# Wait for the workflow to finish
+workflow = wait_for_workflows_to_complete(
+    client=client,
+    workflow_ids=[workflow_id],
+    log_events=["STATE_CHANGES"],
+    timeout=3600,
+)[workflow_id]
+
+# Get the status and Wall-clock time:
+response = client.get(f"v0/workflows/{workflow_id}").json()
+with open(f"{foldername}/{jobname}_status.json", "w") as fp:
+    fp.write(json.dumps(response))
+name = response["name"]
+timetaken = response["duration_seconds"]
+print(f"Workflow {jobname} completed with status: {workflow['status']}")
+print(f"Workflow completed in {timetaken:.2f}s")
+
+# Download results:
+response = client.get(
+    f"/v0/workflows/{workflow_id}/results/download", follow_redirects=True
+)
+with open(f"{foldername}/{jobname}_results.zip", "wb") as fp:
+    fp.write(response.content)
+
+# Extract and print the atomic charges contained in the numeric results:
+response = client.get(f"/v0/workflows/{workflow_id}/results").json()
+with open(f"{foldername}/{jobname}_results.json", "w") as fp:
+    fp.write(json.dumps(response))
+atomic_charges = response["results"]["scf_properties"]["atomic_charges"]
+analysis_methods = [data["analysis_method"] for data in atomic_charges]
+analysis_results = [data["atomic_charges"] for data in atomic_charges]
+
+print()
+print(" Atom |" + " |".join([f"{k:>12s}" for k in analysis_methods]))
+print("------+" + "+".join(["-------------" for k in analysis_methods]))
+for n, vals in enumerate(zip(*analysis_results)):
+    print(f"  {n:3d} |" + " |".join([f"{v:12.8f}" for v in vals]))
+
+# This script will print a table which looks like this:
```

Comment:
Let's note that there could be minor numerical differences

Reply:
Done here, and in the other scripts as well.